src/goto-symex/symex_main.cpp

```diff
@@ -167,6 +167,14 @@ void goto_symext::symex_step(reachability_treet &art)
   case END_FUNCTION:
     symex_end_of_function();
 
+    if (stack_catch.size())
+    {
+      // Get to the correct try (always the last one)
+      goto_symex_statet::exceptiont *except = &stack_catch.top();
```

Comment:
Shall we use `back()` instead of using `top()`?

Reply:
`stack_catch` used a stack container and I think `back()` works for sequence containers.